client/extensions/woocommerce/app/shipping/shipping-zone-list.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ShippingHeader from './shipping-header';
+import ShippingZone from './shipping-zone';
+import ShippingZoneDialog from './shipping-zone-dialog';
+
+class ShippingZoneList extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { showDialog: false };
+
+		this.onAddZoneOpen = this.onAddZoneOpen.bind( this );
+		this.onAddZoneClose = this.onAddZoneClose.bind( this );
+	}
+
+	onAddZoneOpen() {
+		this.setState( { showDialog: true } );
+	}
+
+	onAddZoneClose() {
+		this.setState( { showDialog: false } );
+	}
+
+	render() {
+		const __ = i18n.translate;
+
+		return (
+			<div>
+				<ShippingHeader
+					label={ __( 'Shipping Zones' ) }
+					description={ __( 'Where you will ship and method of which you will ship to' ) }>
```

Comment:
I know this was taken straight from the lo-fi prototype, but I think it could use some wordsmithing.

What about...

> Where you will ship to and the method(s) you'll use to do so.

or..

> Where you'll ship to and the choices you'll give to your customers.

Thought: shipping zones/methods really dictate the shipping experience for customers, not how the merchant has to fulfill the order. (For example flat rate or free shipping - that's not _how_ the method the merchant is using to ship, it's the choice given to the customer.) Perhaps that's too complicated a way of thinking about it.

@kellychoffman @jameskoster